packages/suite/src/views/wallet/coinmarket/common/CoinmarketTransactions/CoinmarketTransaction/CoinmarketTransactionInfo.tsx

```diff
@@ -27,8 +27,10 @@ export const CoinmarketTransactionInfo = ({ trade }: CoinmarketTransactionInfoPr
 
     return (
         <Text margin={{ top: spacings.xs }} variant="tertiary" typographyStyle="label" as="div">
-            {tradeType} • <FormattedDate value={date} date time /> •{' '}
-            <CoinmarketTransactionStatus trade={trade} />
+            <Row alignItems="center" flexWrap="wrap">
```

Comment:
`alignItems="center"` for `Row` is the default value, so you can delete this prop